### DISCUSSION TITLE
TenantDomain

### DISCUSSION THREAD

**Discussion by sirlaugher** [pnp/sp-dev-fx-webparts#5143] [Q&A]
I am getting an error when I do gulp serve on react-tiles. The error is when the page opens I get this URL: https://{tenantdomain}/_layouts/workbench.aspx)/_layouts/workbench.aspx. I do not see a config folder in React-Tiles so I am unsure where to enter the tenantdomain for the sample. I am obviously very new to SPFx and need some guidance on this. I appreciate any help you can provide!! THANKS

**Comment by gretchunkim** [comment]
It's located under config>serve.json file

The config folder is at the top level, same level as src. 

![image](https://github.com/user-attachments/assets/1f587a6f-79cf-4041-80f8-0dfbec5e4909)
